website/docs/cloud-docs/users-teams-organizations/api-tokens.mdx

```diff
@@ -98,3 +98,11 @@ The following chart illustrates the various access levels for the supported API
 | Manage run tasks                   |      🔶     |      🔶     |          🔵          |
 | **Modules**                        |             |             |                     |
 | Manage Terraform modules           |      🔶     | 🔵 (owners) |                     |
+
+## Token Expiration
+
+User, team, and organization tokens may be created with an expiration date and time set. Once this time has passed, the token will no longer be treated as valid, and may not be used to authenticate to any API.
```

Comment:
Non-blocking suggestion: I think we can omit the "set" here. Additionally, what if we explicitly state that API requests will fail if made with an expired token (even though it's implied) -- thoughts?:
```suggestion
User, team, and organization tokens may be created with an expiration date and time. Once this time has passed, the token will no longer be treated as valid, and may not be used to authenticate to any API. Any API requests made with an expired token will result in failure.
```